packages/react/src/components/Switch/Switch.js

```diff
@@ -29,11 +30,11 @@ const Switch = props => {
     onClick({ index, name, text });
   };
 
-  const handleKeyDown = e => {
-    const key = e.key || e.which;
+  const handleKeyDown = event => {
+    const key = event.key;
```

Comment:
Will `key` work in older browsers, or will we need to fallback to `which` or `keyCodes`